stroom-search/stroom-search-extraction/src/main/java/stroom/search/extraction/XPathExtractionElementModule.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2018 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package stroom.search.extraction;
+
+import stroom.pipeline.factory.PipelineElementModule;
+
+class XPathExtractionElementModule extends PipelineElementModule {
```

Comment:
Not sure it needs its own module, just add the bind to the ExtractionElementModule

Reply:
OK